### ISSUE TITLE
IRB or parts of it

### ISSUE THREAD

**Issue by ericgj** [mendicant-original/ruby#8]
There's a lot of good tutorials on IRB, but hardly anything in the code. Do you think it's worth condensing some of what's out there into the docs?  I'm thinking in particular of adding 
- description and usage examples to top-level IRB
- description and usage of some of the more useful of the "extend command bundle" -- irb_load etc. and job (subsession) commands.

Eric

**Comment by semmons99** [comment]
Yeah, I think this would be very handy. I hate telling newcomers to search google instead of reading the docs.

:+1:

**Comment by practicingruby** [comment]
+1
